src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -305,7 +306,11 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef<AppLayoutProps.Ref, AppLa
           navigation={resolvedNavigation && <AppLayoutNavigation appLayoutInternals={appLayoutInternals} />}
           navigationOpen={navigationOpen}
           navigationWidth={navigationWidth}
-          tools={activeDrawer && <AppLayoutDrawer appLayoutInternals={appLayoutInternals} />}
+          tools={
+            ((drawers!.length > 0 && !toolsHide) || activeDrawer) && (
```

Comment:
Need tools to mount even if the drawer is not active (open).

Reply:
That's unfortunate, because I was hoping to make the behavior more straightforward. But if our tests rely on it very much, than our customers do it too and we cannot break them 🥲